src/database/index.tsx

```diff
@@ -545,7 +545,7 @@ export const duplicateScenario = async (
       });
     }
 
-    batch.commit();
+    await batch.commit();
```

Comment:
This fixes the "race condition" which makes a lot of sense.  I wasn't waiting for the commit to finish before trying to load the scenarios and therefore data was missing when trying to hydrate the objects in the transforms.  This is indeed the way other similar methods like `saveFacility` are working:  https://github.com/Recidiviz/covid19-dashboard/blob/master/src/database/index.tsx#L438

Reply:
Great find!